.github/workflows/CI.yml

```
@@ -20,6 +20,7 @@ jobs:
           - "1.4"
           - "1.5"
           - "1.6"
+          - "1.7"
```

Since 1.7 is not released yet, I think it needs to be:

```suggestion
          - "1.7-nightly"
```

---

Interesting. When did that change? Does that mean that "1.7-nightly" will break when 1.7 is released?

---

I don't think it will break, but I believe it will always follow the `release-1.7` branch, so it may still be ahead of the latest official 1.7 release.

---

I think what you want here is either `'~1.7.0-0'` or `'^1.7.0-0'`.

https://github.com/julia-actions/setup-julia/blob/master/README.md#julia-versions

---

Yeah, that should be fine as well. I don't think the difference really matters.

---

> `1.7-nightly` ... This version will only be available during certain phases of the Julia release cycle.

I'll go with `'~1.7.0-0'` to be on the safe side

---

```suggestion
          - "~1.7.0-0"
```